### PR TITLE
fix typings in openmrs package

### PR DESCRIPTION
**Pull request by nmalyschkin**
This type errors hinder @samuelmale from building this project. I can't tell why they don't occur on my machine or in the CI pipeline (although my IDE does mark them for me).

### CODE REVIEW

File: packages/openmrs/src/utils/importmap.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ export function runProject(
 
 export async function mergeImportmap(
   decl: ImportmapDeclaration,
-  imports: Record<string, string>
+  imports: Record<string, string> | false
```
Comment (FlorianRappl) [inline packages/openmrs/src/utils/importmap.ts:79]: This does not look right. I guess it should `| undefined`.

**Comment by FlorianRappl** [comment]
> I can't tell why they don't occur on my machine or in the CI pipeline (although my IDE does mark them for me).

It's different versions of TypeScript / the underyling typings.

File: packages/openmrs/src/utils/importmap.ts (hunk 1 above)
Comment (nmalyschkin) [reply packages/openmrs/src/utils/importmap.ts:79]: Well yes, but [this](https://github.com/openmrs/openmrs-esm-core/blob/a71bf5f7ab8f65d4bfa2dd046ba7ddd6eddb3d01/packages/openmrs/src/cli.ts#L83).
I can make it a ternary operator here, would be cleaner anyway.

Comment (FlorianRappl) [reply packages/openmrs/src/utils/importmap.ts:79]: Good point - thanks!

**Pull request merged**